doc/use_cases/turtlebot3_tutorial.md

```diff
@@ -0,0 +1,239 @@
+# Navigation2 - Turtlebot 3
+
+- [Overview](#overview)
+- [Requirments](#requirements)
+- [Navigation2 with Turtlebot 3 in Gazebo](navigation2-with-turtlebot-3-in-gazebo)
+- [Navigation2 with a Real Turtlebot 3](navigation2-with-a-real-turtlebot-3)
+- [Videos](#videos)
+
+# Overview:
+
+This tutorial shows how to control and navigate Turtlebot 3 using the ROS2 Navigation2. Turtlebot is a low-cost, personal robot kit with open-source software (ROS2). Turtblebot robots are widely supported by the ROS community. You can find many applications and examples of Turtlebot projects on the Internet. You can find more information about Turtlebot3 [here.](http://emanual.robotis.com/docs/en/platform/turtlebot3/overview/)
+
+Links to the robot kits:
+
+- [Turtlebot Burger](http://www.robotis.us/turtlebot-3-burger-us/)
+- [Turtlebot Waffle](http://www.robotis.us/turtlebot-3-waffle-pi/)
+
+This tutorial consists of two parts. In the first part, you will learn how to use Turtlebot 3 robots in summation (Gazebo). And in the second part, you will learn how to control a real Turtlebot Waffle using Navigation2.
+
+```ROS2 Dashing``` and ```Navigation2 Dashing 0.2.4``` are used to create this tutorial.
+
+This tutorial may take about 2-5 hours to complete completely depends on your experience with ROS, robots and what computer system you have.
+
+## Requirements:
+
+- [Install ROS2]([https://index.ros.org/doc/ros2/Installation/](https://index.ros.org/doc/ros2/Installation/))
+
+- Install Navigation2
+
+    - `sudo apt install ros-<ros2-distro>-navigation2`
+
+    - `sudo apt install ros-<ros2-distro>-ros-dashing-nav2-bringup`
+
+- [Build Turtlebot3](http://emanual.robotis.com/docs/en/platform/turtlebot3/ros2/#setup)
+
+- [Install Gazebo](http://gazebosim.org/tutorials?tut=install_ubuntu&cat=install)
+
+## Navigation2 with Turtlebot 3 in Gazebo
+
+### 0- Edit .bashrc
+
+First, make sure you have the following lines in your .bashrc file.
+
+Open a new terminal and open your .bashrc file which is located in your home directory. You can use your favorite text editor.
+
+```
+sudo gedit ~/.bashrc
+```
+
+Add the following lines to the end of the file
+
+```
+source /opt/ros/<ros2_distro>/setup.bash
+source ~/turtlebot3_ws/install/setup.bash
+export TURTLEBOT3_MODEL=waffle
+export GAZEBO_MODEL_PATH=$GAZEBO_MODEL_PATH:~/turtlebot3_ws/src/turtlebot3/turtlebot3_simulations/turtlebot3_gazebo/models
+```
+
+save the file and close it. Whenever we open a new terminal, this script will be executed. 
+
+*Please note that this will  make your terminal's start-up time considerably longer, so when you are done with this tutorial you may want to remove these lines from your .bashrc file.*
+
+### 1- Launch Gazebo
+
+Now, launch Gazebo with the world model, open a new terminal and type
+
+    gazebo --verbose -s libgazebo_ros_init.so /home/<user_name>/turtlebot3_ws/src/turtlebot3/turtlebot3_simulations/turtlebot3_gazebo/worlds/turtlebot3_worlds/waffle.model
```

Comment:
`gazebo --verbose -s libgazebo_ros_init.so /path/to/turtlebot3_ws/...` don't make them put it somewhere specific